openelis/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM tomcat:8-jdk8
+
+RUN apt update \
+  && apt install -y gettext-base
+
+ADD resources/openelis.war /usr/local/tomcat/webapps/
```

Comment:
I have just uploaded the WAR file to **nexus.mekomsolutions.net**. Could you fetch it from URL, rather than providing it as resource?
https://nexus.mekomsolutions.net/repository/maven-releases/org/bahmni/openelis/3.1/openelis-3.1.war